src/navigation/navigationhandler_lua.inl

```diff
@@ -25,30 +25,39 @@
 namespace {
 
 /**
- * Load a navigation state from file. The file should be a lua file returning the
```

Comment:
Ah.. I would suggest that all the changes from my draft PR is removed from this PR, because they will be subject to changes. 

I.e., remove the changes in `navigationhandler_lua.inl` and ``pathnavigator_lua.inl` :)